### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Godswilly**
This is a clone application of archived apple website. In this project, my main aim was to learn how to align elements with float, flex, and grid in CSS. I tried doing it by creating an application that looks like the archived apple.com website.

### CODE REVIEW

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @Godswilly I am Alex your code reviewr for this project :raised_hands: 
God job this far, good use of HTML semantic, Readme file descriptive, Pull Request summary good enough, the commits should've been made more often with smaller chunks of code, for the next project remember small commits with meaningful description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
There are still some issues that are preventing me from approving this project, see the comments bellow :point_down: 
## :stop_sign: Required Changes :construction: :construction_worker: 
- [ ] There is a lot of repeating codes in our CSS file, let us apply the DRY rule. For more info kindly check this [article](https://speckyboy.com/good-bad-css-practices/) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- [ ] The content should fill the entire page you have some space around left
image
- [ ] The four images in the `end container` are overlapping and have a weird dark border due to the fact that you probably took screenshots of those pictures as a shortcut to bypass requirements, you can find the original pictures by download [this zip](https://archive.fo/download/UW4oR.zip) and extract it, you will find all the pictures needed inside the extracted folder.
- [ ] Here the footer has some alignment issues with the copyright text, should align along with the content above it, also the color of the a tags should be line in the original page
### Clone:
image
### Original:
image
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: auto;
+}
```
Comment (santiago-dev96) [inline css/style.css]: This is useless, you are using a CSS reset file.
```suggestion
```

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+a {
+  text-decoration: none;
+  color: blue;
+}
+
+.page {
+  display: flex;
+  flex-flow: column;
```
Comment (santiago-dev96) [inline css/style.css]: Better use this.
```suggestion
  flex-direction: column;
```

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  background-image: url("../assets/main-background.jpg");
+  height: 550px;
+  background-repeat: no-repeat;
+  background-position: bottom;
+}
+
+.nav-bar {
+  background-color: #707070;
```
Comment (santiago-dev96) [inline css/style.css]: If you are going to use a linear gradient as background then this has no purpose.
```suggestion
```

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  height: 35px;
+  border-radius: 3px;
+  margin-top: 40px;
+  max-width: 960px;
+  display: flex;
+  box-shadow: (0, 0, 0, 0.3) 0 1px 2px;
+  flex-flow: row;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
  flex-direction: row;
```

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+.nav-gradiant {
+  background: linear-gradient(180deg, rgba(57, 57, 57, 1) 0%, rgba(57, 57, 57, 1) 48%, rgba(63, 63, 53, 1) 51%, rgba(107, 116, 125, 1) 92%);
+}
+
+.nav-lists {
+  display: flex;
+  align-items: center;
+  min-width: 103px;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
 width: 103px;
```

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+  justify-content: center;
+  border-right: 1px solid grey;
+  font-size: 14px;
+}
+
+.nav-search {
+  border: 2px solid transparent;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
  padding: 2px;
```

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+  padding-left: 1.5em;
+  max-width: 100px;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
  width: 100px;
```

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+  border-radius: 15px;
+  background: linear-gradient(to top, rgb(124, 128, 133) 30%, rgb(97, 80, 80) 87%);
+}
+
+.search-img img {
+  max-height: 15px;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
  height: 15px;
```

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+  position: absolute;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
```

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+  color: white;
+}
+
+.nav-lists a {
+  color: honeydew;
+}
+
+.search-image {
+  height: 15px;
+}
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
```

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+
+.mid-content {
+  display: flex;
+  flex-flow: column;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
  flex-direction: column;
```

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+  margin-top: 30px;
+  align-items: center;
+  color: ivory;
+  font-family: 'Raleway', sans-serif;
+}
+
+.mid-content h1 {
+  padding-top: 10px;
+  font-size: 48px;
+  text-transform: none;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
```

File: css/style.css [hunk 13]
```diff
@@ -0,0 +1,165 @@
...
+  padding-top: 10px;
+  font-size: 34px;
+  max-width: 1000px;
+  text-align: center;
+  margin-top: 25px;
+}
+
+.play {
+  display: flex;
+  flex-flow: row;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
  flex-direction: row;
```

File: css/style.css [hunk 14]
```diff
@@ -0,0 +1,165 @@
...
+.mid-content a:hover {
+  text-decoration: underline;
+}
+
+.end-content {
+  margin-top: 2px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  grid-template-rows: minmax(150px, 200px);
+  position: relative;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
```

File: css/style.css [hunk 15]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+footer {
+  margin: auto;
```
Comment (santiago-dev96) [inline css/style.css]:
```suggestion
```

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required :no_entry_sign:

Your file `css/style.css` is full of bad practices, please implement the required changes and submit another CRR.

### OPTIONAL
- Make the search bar a bit more similar to the original page, it is taking too much width, and it is not being animated on focus; I suggest you to read [about CSS animations](https://developer.mozilla.org/en-US/docs/Web/CSS/animation)

You can find me in slack as **srodrig**, have a nice day! :wave:

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

I would, however, recommend making the `div.end-content` element same width with the `div.page` element before merging. :relaxed: 

   record1

> ### Project specific
> The CSS background should `cover` the whole page irrespective of the screen size. This is a learning opportunity for you to use the background size and position properties.

_**Hint:** We can use [`background-size`](https://developer.mozilla.org/en/docs/Web/CSS/background-size) of `cover`, with a fitting `background-position`._

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**